qiskit_machine_learning/connectors/__init__.py

```diff
@@ -0,0 +1,35 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019, 2021.
```

Comment:
I guess we should try with these dates to have the first date correct. I think just 2021 suffices for this file as its completely new right.

Reply:
done.